src/app/pages/Sandbox/Editor/Workspace/Files/DirectoryEntry/Entry/EntryIcons.js

```diff
@@ -69,23 +69,29 @@ type Props = {
   root: ?boolean,
   error: boolean,
 };
-export default ({
+export default function EntryIcon({
```

Comment:
I think it would be cleaner if we'd use the existing way, but do `isOpen = false` in the prop arguments.

Reply:
I've tried with
```js
export default function EntryIcon({
  ...
  isOpen = true,
  ...
: Props) { ...
```
, but I still get `EsLint: propType "isOpen" is not required, but has no corresponding defaultProp declaration. (react/require-default-props)`.

Reply:
Ah, just read this from [here](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md): 

> The same also holds true for stateless functional components: default function parameters do not behave the same as defaultProps and thus using defaultProps is still preferred.

Let's use `defaultProps` then.